spec/capybara_helper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Capybara integration
+require 'capybara/rspec'
+require 'capybara/rails'
+
+Capybara.configure do |config|
+  config.default_max_wait_time = 10
+  config.default_selector = :css
+end
+
+if ENV['CAPYBARA_DRIVER'] == 'selenium'
+  require 'selenium-webdriver'
+
+  Capybara.server = :webrick
```

Comment:
Let's use Puma here. It's already added to the test group bundle by this PR, and that's what we use elsewhere so best to test against it.